pkg/maps/ctmap/ctmap.go

```diff
@@ -350,3 +350,21 @@ func GC(m *bpf.Map, mapName string, filter *GCFilter) int {
 		return 0
 	}
 }
+
+// Flush runs garbage collection for map m with the name mapName, deleting all
+// entries that are due to expire in the next ten minutes.
+func Flush(m *bpf.Map, mapName string) int {
```

Comment:
Add a comment that the map should be opened before this function is called.

Reply:
Thanks for the review, I'll work on this feedback.